PythonPonies/bangazon-api
# Zgl migrate

# Description
Add description about what this pull request covers.
- fixes to api and serializers
## Number of Fixes
Type of number of fixes included in this PR.

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#1 
## Problem to Solve
Describe the problem(s) this pull request solves.
allows customer endpoint to pull in user information
## Proposed Changes
Describe the proposed changes.
Now can navigate to endpoints and get data
## Expected Behavior
Describe the expected behavior.
everything should work
## Steps to Test Solution

1. git fetch
2. python runserver
3. everything should work and should see some instances of items except customers and users

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
